metals/src/main/scala/scala/meta/internal/metals/ClientExperimentalCapabilities.scala

```diff
@@ -4,37 +4,45 @@ import org.eclipse.{lsp4j => l}
 import com.google.gson.JsonNull
 
 final case class ClientExperimentalCapabilities(
-    debuggingProvider: java.lang.Boolean = false,
-    treeViewProvider: java.lang.Boolean = false,
-    decorationProvider: java.lang.Boolean = false,
-    inputBoxProvider: java.lang.Boolean = false,
-    quickPickProvider: java.lang.Boolean = false,
-    didFocusProvider: java.lang.Boolean = false,
-    slowTaskProvider: java.lang.Boolean = false,
-    executeClientCommandProvider: java.lang.Boolean = false,
-    openFilesOnRenameProvider: java.lang.Boolean = false,
-    doctorProvider: String = "html",
-    statusBarProvider: String = "off"
+    debuggingProvider: Option[Boolean] = Some(false),
```

Comment:
I believe defaults `None` is better semantically to match the `null` from Gson, this way in `ClientConfiguration` we can better merge values, especially the none Boolean ones

Reply:
Ahh good point. I'll change this... if I can actually get the `null`s to be turned into `None`s...